src/components/general/world-objects-list/WorldObjectsList.jsx

```diff
@@ -76,7 +79,16 @@ export const WorldObjectsList = ({ app, opened, setOpened }) => {
 
             case 90: {   // Z
 
-                setOpened( ! opened );
+                if ( opened ) {
```

Comment:
Keys are handled in the IO manager. I don't think React should handle key events.